playbooks/files/delete_cts_tracker.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python3
+
+import openstack
+
+conn = openstack.connect()
+conn.cts.delete_tracker()
```

Comment:
Is the system tracker the default? Or will any tracker be deleted if it is there?